extension/src/popups/MinervaMenu.js

```diff
@@ -1,6 +1,8 @@
-import { renderStudentRankings } from "./MinervaMenuRenderInfo.js";
+import { filterAvailableStoreStickers, renderCurrentStudentInfo, renderStudentRankings } from "./MinervaMenuRenderInfo.js";
+import { getStudentHandle } from "../utils/ApiInterface.js";
 
 function transitionToMainMenu(studentName) {
+    diplayCurrentStudentInfo(studentName);
```

Comment:
Let's not pass the student name here

Reply:
You want to call getStudentHandle repeatedly?

Reply:
Or do you want to use globals. I tend to stay away from globals.

Reply:
I was thinking ahead on #97. For this pull request, I guess you have to do it this way.

The way I envision this file in terms of purpose is to check for login and then offset all messaging/api stuff into the render file. It can stay what it is for now.